.circleci/config.yml

```diff
@@ -55,15 +55,15 @@ executors:
 
   machine_executor_arm64:
     machine:
-      image: ubuntu-2004:202201-02 # Ubuntu 20.04, Docker v20.10.12, Docker Compose v1.29.2
+      image: ubuntu-2004:current # https://circleci.com/developer/machine/image/ubuntu-2004
     resource_class: arm.medium
     environment:
       architecture: "arm64"
       platform: "linux/arm64"
 
   machine_executor_amd64:
     machine:
-      image: ubuntu-2004:202201-02 # Ubuntu 20.04, Docker v20.10.12, Docker Compose v1.29.2
+      image: ubuntu-2004:current # https://circleci.com/developer/machine/image/ubuntu-2004
```

Comment:
We are using quite an old image, ubuntu-2204 would be better to use

Reply:
👍🏼 sure. easy change to make.